pages/ballot/index.tsx

```diff
@@ -1,34 +1,71 @@
-import { Layout, LoaderFlag } from "components";
+import { useState } from "react";
 import { GetServerSideProps, NextPage } from "next";
 import Head from "next/head";
-import styles from "components/Layout/Layout.module.scss";
+import { useRouter } from "next/router";
+import { dehydrate, QueryClient } from "react-query";
+
+import {
+  Layout,
+  LoaderFlag,
+  FlagSection,
+  VotingGuideWelcome,
+  OfficeRaces,
+} from "components";
+
 import {
   PoliticalScope,
   RaceResult,
   useUpcomingElectionsQuery,
+  useVotingGuideByIdQuery,
 } from "generated";
-import { FlagSection } from "components";
+
 import { dateString } from "utils/dates";
 import { groupBy } from "utils/groupBy";
-import { dehydrate, QueryClient } from "react-query";
+
 import { useAuth } from "hooks/useAuth";
-import { VotingGuideWelcome } from "components/VotingGuide/VotingGuideWelcome";
-import { useState } from "react";
 import { VotingGuideProvider } from "hooks/useVotingGuide";
-import { OfficeRaces } from "components/Ballot/OfficeRaces";
+import { useSavedGuideIds } from "hooks/useSavedGuideIds";
+
+import { VOTING_GUIDE_WELCOME_VISIBLE } from "utils/constants";
+
+import styles from "components/Layout/Layout.module.scss";
 
 const BallotPage: NextPage<{ mobileNavTitle?: string }> = ({
```

Comment:
This will need to use `NextPageWithLayout` for consistency.

Reply:
Definitely out of scope for this work but props to you if you end up updating here!

Reply:
I'm going to leave this alone for now, I didn't make any changes to this area. But would be a good isolated task